.github/workflows/build.yml

```diff
@@ -21,10 +21,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           repository: opensearch-project/OpenSearch
```

Comment:
one more nit, we could nuke checking out OpenSearch :)

Reply:
Done